{"name": "用户手机登入", "status": "passed", "attachments": [{"name": "stdout", "source": "ea9be745-92ee-48c4-98d2-a89d8b98d243-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "datas", "value": "{'username': 15397919589, 'password': 123456, 'expected': '无法使用提供的认证信息登录'}"}], "start": 1671380867901, "stop": 1671380868013, "uuid": "84ab9029-ca8d-41f1-82a5-79d75bfd84ec", "historyId": "7f7055b0a55546d465faf54e6155b267", "testCaseId": "7f5d9b461645abeb9a19f6ec1503a4c7", "fullName": "test_case.usercenter.test_user.TestUser#test_login", "labels": [{"name": "feature", "value": "用户中心模块"}, {"name": "story", "value": "用户登入"}, {"name": "parentSuite", "value": "test_case.usercenter"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "xiaoma"}, {"name": "thread", "value": "303052-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.usercenter.test_user"}]}